### PR TITLE
Add `.ipynb` script documentation

### PR DESCRIPTION
**Pull request by mederka**

### CODE REVIEW

File: docsite/docs/Docs/fal-cli/notebook-files.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+`init_fal` requires three inline arguments:
+
+- `project_dir`: path to the dbt project directory
+- `profiles_dir`: path to the dbt profiles directory
+- `default_model_name`: the model name that this notebook is attached to in notebook runtime.
+
+Once executed, you can use fal functions in your notebook's Python cells:
+
+```python
+my_df = ref('my_model')
```
Comment (chamini2) [inline docsite/docs/Docs/fal-cli/notebook-files.md:32]: Change the model name so it does not get confused with the `default_model_name` one passed in

File: docsite/docs/Docs/fal-cli/notebook-files.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+write_to_model(my_predictions)
```
Comment (chamini2) [inline docsite/docs/Docs/fal-cli/notebook-files.md:34]: Add a `# ...` or something before this, `my_predictions` variable does not exist in the scope

File: docsite/docs/Docs/fal-cli/notebook-files.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+%init_fal project_dir=project_dir profiles_dir=profiles_dir default_model_name=my_model
+```
+
+`init_fal` requires three inline arguments:
+
+- `project_dir`: path to the dbt project directory
+- `profiles_dir`: path to the dbt profiles directory
+- `default_model_name`: the model name that this notebook is attached to in notebook runtime.
```
Comment (chamini2) [inline docsite/docs/Docs/fal-cli/notebook-files.md:27]: Add a note this only applying to `write_to_model`

File: docsite/docs/Docs/fal-cli/notebook-files.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Once executed, you can use fal functions in your notebook's Python cells:
+
+```python
+my_df = ref('my_model')
+
+write_to_model(my_predictions)
+```
+
+Note that the `default_model_name` is only active notebook runtime. When run using `fal run` or `fal flow run`, fal will determine the model according to the relevant `schema.yml` file. In fal runtime `init_fal` arguments are ignored. You can specify a .ipynb file the same way as a regular Python file:
```
Comment (chamini2) [inline docsite/docs/Docs/fal-cli/notebook-files.md:37]:
```suggestion
Note that the `default_model_name` is only active during notebook runtime. When run using `fal run` or `fal flow run`, fal will determine the model to write to according to the relevant `schema.yml` file. In the fal runtime, the `init_fal` line is ignored. 

You can specify a .ipynb file the same way as a regular Python file:
```

File: docsite/docs/Docs/python-package.md [hunk 5]
```diff
@@ -28,12 +28,6 @@ tickets = faldbt.ref('stg_zendesk_ticket_data')
 # pandas.DataFrame
 ```
 
-##  Interacting with dbt from a Notebook
```
Comment (chamini2) [inline docsite/docs/Docs/python-package.md:31]: Should we delete this entirely or link to the new doc?

Comment (mederka) [reply docsite/docs/Docs/python-package.md:31]: I think we can delete it as it's not so relevant given `init_fal`.

**Review by chamini2 — APPROVED** [review]
Cool!

**Pull request merged**